### PR TITLE
Pgb 1370/hr request

### PR DESCRIPTION
**Pull request by JKoetsier**

### CODE REVIEW

**Comment by Vegter** [comment]
Some modules seem to be missing?

**Comment by JKoetsier** [comment]
> Some modules seem to be missing?

Si. And the message thread wasn't starting from uwsgi yet. All is fixed now :)

File: src/tests/hr/kvk_dataservice/test_binary_signature.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import freezegun
```
Comment (Vegter) [inline src/tests/hr/kvk_dataservice/test_binary_signature.py:1]: nice

**Review by Vegter — APPROVED** [review]
mooie opzet

**Pull request merged**